Fits/Dreadnoughts/[FL33T] Anti-Cap Revelation V2.md

```diff
@@ -8,43 +8,44 @@ Used to provide high DPS against other capital ships.  Highly ineffective for us
 
 ## Fit
 ```
-[Revelation, T2 Anti-Cap Revelation]
+[Revelation, Anti-Cap Revelation V2]
 
-Damage Control II
 25000mm Rolled Tungsten Compact Plates
-25000mm Rolled Tungsten Compact Plates
-Multispectrum Energized Membrane II
-Multispectrum Energized Membrane II
+25000mm Crystalline Carbonide Restrained Plates
+Corpus A-Type Thermal Armor Hardener
```

Comment:
Looking at our previous losses, we're taking damage from a ton of sources. https://zkillboard.com/kill/114572067/

A-Type multispectrums seem to be a better fit for these cases, and also what a large majority of other groups use. Thoughts?

Also, T2 plates might be worth it here, and we aren't really getting much from the imperial navy sink.

Reply:
Also, we aren't heavily multi-boxing dreads like other groups. Flex hardener may give us some value?

Reply:
A-Types are just expensive, it's not a bad increase in EHP but 800mil a pop they felt a bit out of the cost structure we were viewing these through. That would basically be a 30% increase in the fitting cost (-hull).

Imp Navy Sink was purely for fitting, and T2 plates are much harder on fitting and not super big stat increases while also being very expensive for what they are, so again it's kind of a cost-effectiveness question.

Flex hardeners are just a lot of cap, the general response was that the cap drain was too big on ships that were already being a bit skimpy on cap all things considered.

Reply:
Looking at the A-Type part though, I am all for putting those on the RevNavy/MorosNavy/Zirnitra. The effect is way more pronounced and the cost is much more in-line with the rest of the ship.

Reply:
Flex hardeners' cycle time makes them pretty useless in any situation where you don't know exactly what damage type you're going up against. Some groups have them permanently set to fill a resist hole (Like explosive) but if incoming damage isn't in that profile its just a wasted slot. Looking at that KM, >90% of the incoming damage was from other Revs/Zirn. Those are going to be the two most common dreads that we go up against. 

Also, like Mort said with the Rev Navy, T2 plates really don't provide very much benefit for their cost. 

For the A-type membrane, it really comes down to how expensive we want the dreads to be. My goal was to keep the cost around 5.5b (with the hull being 3.5b). Right now we're around 5.8b.

Reply:
Benefit on the Rev is that the A-Type coating lets you debling the Heat Sink and Crystalline the last plate so the cost is more or less negligible on that one. Moros I agree it's not worth it on.